alert.css

```diff
@@ -0,0 +1,115 @@
+header{
+
+        display: flex;
+        justify-content: space-between;
+        margin: 5px;
+        width: 100%;
+        background-color: rgb(155, 173, 238);
+        border-bottom-left-radius: 1rem;
+        border-bottom-right-radius: 1rem;
+        padding:1px;
+        top: 0;
+        /* position: sticky; */
+
+}
+
+.header{
+    text-align: center;
+}
+
+.container{
+    display: flex;
+}
+
+
+.side-nav{
+    text-decoration: none;
+}
+
+.side-nav:hover{
+    background-color: turquoise;
+    padding: .2rem 2rem .2rem 2rem;
+    border-radius: 1rem;
+}
+
+
+.sidebar{
+    width: 27rem;
+    height: fit-content;
+    border-right-style: solid;
+    /* border: 1px solid black; */
+    position: static;
+    text-align: center;
+}
+
+
+
+.alert{
+    display: inline-block;
+    margin: 1rem;
+    width: 100%;
+    padding: .7rem;
+    text-align: center;
+}
+.btn{
+    /* width: 10rem; */
+    margin-top: 30px;
+    display: flex;
+    justify-content: space-between;
+}
+
+button{
+    border-radius: 1rem;
+    padding: 10px;
+    background-color: skyblue;
+}
+
+.icon{
+    font-size: 30px;
+    cursor: pointer;
+}
+
+.alert1{
+    background-color: skyblue;
+}
+
+.alert2{
+    background-color: rgb(186, 191, 93);
+}
+
+.alert3{
+    background-color: lightcyan;
+}
+
+.alert4{
+    background-color: red;
+}
+
+.alert5{
+    background-color: lightgoldenrodyellow;
+}
+
+.alert6{
+    background-color: lightskyblue;
+}
+
+.alert7{
+    color: grey;
+}
+
+.alert8{
+    background-color: grey;
+}
```

Comment:
Suggestion: instead of number sequence you can give class names with the color e.g alert-red, alert-yellow, alert-blue etc... like that